docs/getting-started.md

```diff
@@ -4,32 +4,63 @@ title: Getting Started
+program is (potentially) running on many nodes at all: There is no notion of
+_ranks_; **partitioning of work and data is taken care of transparently behind the
+scenes**. This lets you focus on your actual work, without having to concern
+yourself with the complexities of modern distributed memory cluster programming.
+
+### A Word of Caution
+
+While ease of use is one of Celerity's main goals, simplicity can only go so far
+without sacrificing considerable performance. **Proficiency in modern C++ as
+well as at least a rough understanding of how accelerator (GPU) programming
+differs from parallel CPU programming is required** to make efficient use of
+Celerity. Additionally, you will require a good understanding of the algorithms
+and techniques you intend to implement using Celerity in order for the runtime
+system to be able to run it on a cluster both correctly and in an efficient
+manner.
+
+## Built on a Strong Foundation
 
 Celerity is built on top of [SYCL](https://www.khronos.org/sycl/), an
-open-standard high-level C++ embedded domain specific language for
-programming accelerators. SYCL provides a great API that hits a sweet spot
-between expressiveness and power as well as ease of use, making it the
-perfect starting point for Celerity: We set out to find the minimal set of
-extensions required to bring the SYCL API to distributed memory clusters -
-thus making it relatively easy to migrate an existing SYCL application to
-Celerity. If you don't have any experience with SYCL, don't worry, as we will
-introduce the most important concepts along the way.
+open-standard high-level C++ embedded domain specific language for programming
+accelerators. SYCL provides a great API that hits a sweet spot between
+expressiveness and power as well as ease of use, making it the perfect starting
+point for Celerity: We set out to find the **minimal set of API extensions**
+required to bring the SYCL API to distributed memory clusters - thus making it
+relatively **easy to migrate an existing SYCL application to Celerity**. If you
+don't have any experience with SYCL, don't worry, as we will introduce the most
+important concepts along the way.
+
+### Points of Divergence
+
+While it is one of Celerity's [Core Principles](core-principles.md) to stick as
+closely to SYCL as possible, **the Celerity API is neither a super- nor subset
+of the SYCL API**: On one hand certain SYCL features, such as SYCL 2020's
+unified shared memory (USM), are inherently unsuitable for distributed memory
+execution and can therefore not be supported by Celerity. On the other hand,
+certain high-performance computing (HPC) features such as [Collective Host
+Tasks](host-tasks.md#experimental-collective-host-tasks) are required to make
+Celerity work at scale.
+
+> Starting with Celerity 0.3.0, all supported SYCL features are made available
+> through the `celerity::` namespace, in addition to the `sycl::` namespace.
```

Comment:
There's still `sycl::vec`, `sycl::float4`, `sycl::plus` among others which we support. We only re-export / re-implement types that Celerity interacts with explicitly, and leave things that are transparent to Celerity in `sycl::`. I wonder if we can find a description for this that is less focused on our implementation details.

Reply:
Due to a lack of creativity on my part we now just say "most supported SYCL features". Need to clarify at some (future) point.